grease/src/Grease/Refine/Diagnostic.hs

```diff
@@ -124,8 +126,9 @@ instance PP.Pretty Diagnostic where
           , PP.indent 4 (C.ppSimError $ lp ^. W4.labeledPredMsg)
           , -- , PP.indent 4 (W4.ppExpr $ lp ^. W4.labeledPred)
             case minfo of
+              Just (MacawMemError (UnmappedGlobalMemoryAccess _)) -> "<Unmapped memory>"
```

Comment:
Why the angle brackets? Also, should we perhaps print the bitvector? In fact, should there maybe be a `Pretty` instance upstream, so that we could just call `pretty` here?